app/src/main/java/com/github/globant/githubsubscribers/commons/utils/Constants.java

```diff
@@ -12,6 +12,7 @@
 
     public static final String DEBUG_PREFIX = "GITHUBAPI";
     public static final boolean DEBUG_LOGS = true;
+    public final static String MESSAGE_FAILED_SERVICE = "The sevice is not available";
```

Comment:
Please, use string resources for this kind of messages.

If we wanted to support multiple languages as Spanish, French, Chinese, etc. How would you manage it into a constant?